src/ducks/Watchlists/Widgets/TopBar/TopBar.module.scss

```diff
@@ -47,3 +47,65 @@
   align-items: center;
   justify-content: space-between;
 }
+
+.commentsWrapper {
+  background-color: var(--white);
+  border-top: 1px solid var(--porcelain);
+  border-top-left-radius: 4px;
+  border-left: 1px solid var(--porcelain);
+  border-bottom-left-radius: 4px;
+  width: 363px;
+  z-index: 11;
+  position: fixed;
+  top: var(--screener-top-panel-height);
+  bottom: 0;
+  right: 0;
+  transition: transform 0.3s ease-out;
+  transform: scaleX(0);
+  transform-origin: top right;
+  user-select: none;
+  padding: 16px 24px;
+}
+
+.active {
+  transform: scaleX(1);
+  animation: growPartially 0.35s ease-out;
+}
+
+.background {
+  position: fixed;
+  z-index: 9;
+  background-color: $rhino;
+  opacity: 0.6;
+  top: var(--screener-top-panel-height);
+  left: 0;
+  right: 0;
+  bottom: 0;
+  animation: appearPartially 0.4s ease-in-out;
+}
+
+.closeWrapper {
+  --bg: var(--white);
+  --bg-hover: var(--white);
+  --fill: var(--waterloo);
+  --color: var(--waterloo);
+  --fill-hover: var(--jungle-green);
+  --color-hover: var(--jungle-green);
```

Comment:
`.closeWrapper` contains an `svg` element, therefore only `fill` option should be adjust.
Use `var(--green)` which is up-to-date [webkit](https://github.com/santiment/san-webkit/blob/master/src/styles/color.css#L1)'s color
`background` should be transparent by default. If it's not set only `--bg` variable.
`cursor: pointer` rule is redundant.

In conclusion:
```css
.closeWrapper {
  --bg: var(--white);
  --fill: var(--waterloo);
  --fill-hover: var(--green)
}
```